selftests/checkall

```diff
@@ -115,7 +115,8 @@ if [ "$TRAVIS" == "true" ]; then
 else
     run_rc lint 'inspekt lint --exclude=.git --enable W0102,W0611'
 fi
-run_rc indent 'inspekt indent --exclude=.git'
+# Skip checking test_utils_cpu.py due to inspektor bug
```

Comment:
Do we have a way to monitor the bug status? If yes, please include the check in `selftests/functional/test_thirdparty_bugs.py`

Reply:
Yep, good point.  This is also tracked in GitHub.  Will add it.